recipe/meta.yaml

```diff
@@ -10,31 +10,103 @@ source:
   url: https://epics.anl.gov/download/base/base-{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - config_site.diff  # [not win]
-    - config_common.diff  # [not win]
-    - config_readline.diff  # [linux]
+    - CONFIG_COMMON_install_permissions.patch
+    - force_readline_commandline_library.patch  # [linux]
 
 build:
   number: 0
+  run_exports:
+    - {{ pin_subpackage('epics-base', max_pin='x.x.x.x') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make  # [not win]
-    - m2w64-make  # [win]
-    - perl  # [win]
+    - make
+    - perl
+    - m2-patch      # [win]
+    # To run pre-build.py
+    - python >=3.6  # [win]
+  host:
+    - readline      # [linux]
 
 test:
+  source_files:
+    - modules/pva2pva/loopback.conf
   commands:
-    - test -f ${PREFIX}/epics/bin/${EPICS_HOST_ARCH}/caget  # [not win]
-    - test -f ${PREFIX}/epics/include/epicsVersion.h  # [not win]
-    - test -f ${PREFIX}/epics/lib/${EPICS_HOST_ARCH}/libCom${SHLIB_EXT}  # [not win]
-    - if %ARCH%==32 (set EPICS_HOST_ARCH=win32-x86) else if %ARCH%==64 (set EPICS_HOST_ARCH=windows-x64)  # [win]
-    - if not exist %PREFIX%\epics\bin\%EPICS_HOST_ARCH%\caget.exe exit 1  # [win]
-    - if not exist %PREFIX%\epics\bin\%EPICS_HOST_ARCH%\Com.dll exit 1  # [win]
-    - if not exist %PREFIX%\epics\lib\%EPICS_HOST_ARCH%\Com.lib exit 1  # [win]
-    - if not exist %PREFIX%\epics\include\epicsVersion.h exit 1  # [win]
+    - caget -h
+    - cainfo -h
+    - caput -h
+    - camonitor -h
+    - pvget -h
+    - pvput -h
+    - pvinfo -h
+    - pvlist -h
+    - softIoc -h
+    - softIocPVA -h
+    # p2p -h exits with 1... Use -C flag to test it instead
+    - p2p -C modules/pva2pva/loopback.conf
```

Comment:
For `p2p`, it is my understanding that it is being replaced by the new `pvagw` available with `p4p` so I'm not sure if that is required here.
The test list looks very complete and great! 👍

Reply:
Thanks. True about p4p. The p2p test works right now, so I propose to leave it. We can remove it later.